jobbergate-docs/src/developer_guide/integration_testing.rst

```diff
@@ -0,0 +1,431 @@
+=====================
+ Integration Testing
+=====================
+
+When you integration testi Jobbergate, it is essential that you test the full cycle of
+the platfrom from Application creation to remote Job Submission through the Cluster
+Agent.
+
+For most changes, it is adequate for you to use the ``docker-compose`` configuration
+found in the `Jobbergate Composed <https://github.com/omnivector-solutions/jobbergate/jobbergate-composed>`_
+subproject. Start with the guide on that subproject's README. Take note of how to
+exeucte the Jobbergate CLI commands as they will be used extensively in the intergration
+testing. You may use the automatically created user for testing
+(username: "local-user", password: "local").
+
+The integration testing should cover the following work-flows:
+
+* Logging in through the CLI
+* Creating an Application
+* Querying a single Application
+* Updating an Application
+* Rendering a Job Script from an Application
+* Updating a Job Script
+* Submitting a Job Script
+* Verifying a Job Submission
+* Deleting the Job Submission, Job Script, and Application
+* Logging out through the CLI
+
+
+Setup
+-----
+
+First, start up the Jobbergate system in docker-compose. Navigate to the
+``jobbergate-composed`` subproject and run the following command:
+
+   .. code-block:: console
```

Comment:
No need for indentation on this one, it is leading to unexpected empty blocks. Please, change all the other occurrences on this page.

```suggestion
.. code-block:: console
```

Reply:
Embarassing! Fixed in commit d2ca89e.